src/design-tokens/tier-1-definitions/typography.json

```diff
@@ -41,7 +41,7 @@
         "value": "0.75rem"
       },
       "11": {
-        "value": "0.688rem"
+        "value": "0.6875rem"
```

Comment:
Adding precision so that the pixel size doesn't include any partial pixels

Reply:
Somehow, our process won't take changes like this, and generate other diffs to be committed (re: #1608)